app/components/shared/map_location_component.rb

```diff
@@ -58,6 +59,7 @@ def data
         marker_investments_coordinates: investments_coordinates,
         marker_latitude: map_location.latitude.presence,
         marker_longitude: map_location.longitude.presence,
+        marker_clustering: feature?("map.marker_clustering"),
```

Comment:
I intended to use `feature` in the key name, but I removed it accidentally at some point. I know we no longer need the `feature` part in the setting key name, but I thought adding it would make it easier for developers to recognise the kind of setting just by reading its key name. This was also why I modified the existing `feature?` helper method instead of adding a new one so developers use the same helper for all "boolean" settings.

I do not have a strong preference. I just wanted to share the reasoning behind my proposal, just in case you like it. IMO, both ways have inconveniences. I don't know which approach is the better one.

Reply:
If I understand correctly, there are two questions here:

1. Whether to rename `map.marker_clustering` to `map.feature.marker_clustering`
2. If we keep `map.marker_clustering`, whether to change the `feature?` helper

Regarding the first question, right now boolean settings right now either start with `feature` or don't contain the word `feature` at all, although it's true that we've never had a case where we mix boolean settings and text settings (like we're doing in `map.*` settings now) :thinking:.

So no strong preference here.

Regarding the second question, until now we expected the `feature?` helper to deal with settings containing `feature` (or `process` :relieved:) in their names. Maybe it's because I'm used to it, but I think I prefer it this way :thinking:.